### DISCUSSION TITLE
How to remove not accessibal encrypted devices?

### DISCUSSION THREAD

**Discussion by Mannshoch** [linuxmint/discussions#157] [Questions]
Im Family admin and try to migrate away from Ubuntu. Cinnamon is planed to be used on a Convertible Laptop. User has no Computer knowledge. Currently, Nemo shows three not accessible devices. 
![Cinamon-devices](https://github.com/linuxmint/discussions/assets/7795891/a764277b-a1f0-4179-8111-9a35f8b1b85d)
How could I get rid of them?

**Comment by rcalixte** [comment]
Can you see if [this forum post](https://forums.linuxmint.com/viewtopic.php?t=119295) works for you? The suggestion was to mark the drives not to be mounted automatically via Disks. (You may need to modify /etc/fstab if not.)

**Reply by Mannshoch** [reply]
Thanks a lot I will test this.
Should I create an Issue for that?
